SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -18,4 +21,12 @@ public ConferenceServiceImpl(ConferenceRepository conferenceRepository) {
         return conferenceRepository.findAll();
     }
 
+    @Override
+    public Conference findConferenceById(Long conferenceId) {
+        Optional<Conference> conference = conferenceRepository.findById(conferenceId);
+        if(conference.isPresent())
+            return conference.get();
+
+        return null;
```

Comment:
Do not return null. Generally it is a good practice to return an Optional for single object and Collection.empty() for list/sets.